.github/workflows/release.yml

```diff
@@ -1,4 +1,6 @@
-name: changesets
+# NOTE: This filename must be called release.yml. If changed, update the Trusted Publisher settings in npm
+
+name: Release
```

Comment:
would this create a conflict since there exists a job jobs.release named Release as well. (Non blocking btw)

Reply:
I think it is fine, as this is just used for reporting. The cypher-builder has the same setup and seems to be working fine :thinking: